packaging/os/yaourt.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/python -tt
+# -*- coding: utf-8 -*-
+
+# (c) 2012, Afterburn <http://github.com/afterburn>
+# (c) 2013, Aaron Bull Schaefer <aaron@elasticdog.com>
+# (c) 2014, RubenKelevra <ruben@vfn-nrw.de>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: yaourt
+short_description: Manage packages with I(yaourt)
+description:
+    - Manage packages with the I(yaourt) package manager, an wrapper around pacman, 
+      which is used on Arch Linux and its variants to access the AUR.
+version_added: "1.0"
+author: Afterburn
+notes: []
+requirements: []
+options:
+    name:
+        description:
+            - Name of the package to install, upgrade, or remove.
+        required: false
+        default: null
+
+    state:
+        description:
+            - Desired state of the package.
+        required: false
+        default: "present"
+        choices: ["present", "absent", "latest"]
+
+    recurse:
+        description:
+            - When removing a package, also remove its dependencies, provided
+              that they are not required by other packages and were not
+              explicitly installed by a user.
+        required: false
+        default: "no"
+        choices: ["yes", "no"]
+        version_added: "1.3"
+
+    force:
+        description:
+            - Force remove package, without any checks.
+        required: false
+        default: "no"
+        choices: ["yes", "no"]
+        version_added: "2.0"
```

Comment:
no need to add version_added to every option when it is the same as when module was added